PROTO_tests/tests/pdarray_creation_test.py

```diff
@@ -658,6 +668,11 @@ def test_fill(self, size, dtype):
         ones.fill(dtype(2))
         assert (dtype(2) == ones).all()
 
+        # Test that int_scalars covers uint8, uint16, uint32
+        ones.fill(np.uint8(2))
+        ones.fill(np.uint16(2))
+        ones.fill(np.uint32(2))
```

Comment:
I think if we instantiate an array we should check the output is reasonable.

Reply:
Same response as above.